src/v/utils/bytes_hashers.h

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2020 Vectorized, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+
+#pragma once
+#include "bytes/bytes.h"
+#include "hashing/xx.h"
+
+struct xx64_bytes_hasher {
```

Comment:
we don't need this. you can define this hwere youuse it. see this

https://github.com/vectorizedio/redpanda/blob/dev/src/v/static_deleter_fn.h

for example. is just a c function, so not sure why we need to define a wrapper per function?